pkg/build/test.go

```diff
@@ -426,6 +426,10 @@ func (t *Test) TestPackage(ctx context.Context) error {
 		return fmt.Errorf("unable to build workspace config: %w", err)
 	}
 
+	if _, ok := cfg.Environment["HOME"]; !ok {
+		cfg.Environment["HOME"] = "/root"
+	}
+
```

Comment:
Can we do this inside of `buildWorkspaceConfig` instead?

Reply:
ack. yeah, that does seem better.